packages/devtools_app/lib/src/screens/network/har_network_data.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/foundation.dart';
```

Comment:
please add copyright header like other pages have with the 2024 year

Reply:
done